### DISCUSSION TITLE
In Automation, how to handle the quick assist dialog window using Appium commands.

### DISCUSSION THREAD

**Discussion by yukender** [appium/appium#18098] [General]
When a case is triggered and run via sauce-lab environment, the android tablet devices enables the quick assist window and the object behind the screen cannot be viewed and the Appium does not recognize the object behind the window. Is there any solution to handle this window using Appium commands. 

![Screenshot of quick assist bar](https://user-images.githubusercontent.com/62298454/213993418-629e9c29-ab26-4c3b-b3bc-ec66ed333cb3.png)

**Comment by jlipps** [comment]
I would reach out to Sauce Labs support if it only happens there.

**Reply by yukender** [reply]
Is there any appium commands to address this issue?

**Reply by jlipps** [reply]
There are no special appium commands no. Can you interact with those items as regular elements? Do they appear in the UI hierarchy? How would a user get rid of that window? Think of strategies like that to address.

**Reply by yukender** [reply]
We cannot interact with those items as regular elements and it is a sauce lab device cleanup issue.
